docs/source/development/commands_reference.md

```diff
@@ -491,7 +491,7 @@ To start an IPython shell:
 kedro ipython
 ```
 
-Every time you start or restart a notebook kernel, a startup script (`<project-root>/.ipython/profile_default/startup/00-kedro-init.py`) will add the following variables in scope:
+Every time you start or restart a notebook kernel, a startup script (`<project_root>/.ipython/profile_default/startup/00-kedro-init.py`) will add the following variables in scope:
```

Comment:
This section is way out of date. No need to deal with it here but let's remember to fix this.

Reply:
I did this in https://github.com/kedro-org/kedro/pull/1761.